course/templates/course/index.html

```diff
@@ -12,85 +12,148 @@ <h1>{{ welcome_text|safe }}</h1>
 </div>
 {% endif %}
 
-<div class="row course-thumbnails">
-    {% site_advert as advert %}
-    {% if advert %}
+{% if my_instances %}
+<div class="frontpage-panel">
+    <h2 class="panel-heading">{% trans "My courses" %}</h2>
+    <div class="row frontpage">
+        {% for instance in my_instances %}
+        {% with instance_url=instance|url %}
         <div class="col-sm-6 col-md-4 col-lg-3">
-            <div class="thumbnail advert-thumbnail">
-                <div class="image" aria-hidden="true">
-                    {% if advert.image %}
-                        <img src="{{ advert.image }}" alt="">
-                    {% else %}
-                        <h1 class="glyphicon glyphicon-comment"></h1>
-                    {% endif %}
+            <div class="frontpage-course card">
```

Comment:
As previously discussed, it would possibly make sense if the whole card was clickable - similar to, for example, the news items on the aalto.fi homepage. It's much nicer for touch navigation and possibly the expected behaviour given the panel size. 

How about making this an issue to revisit later, as it's not necessary to change now?

Reply:
Hmm. Aalto.fi actually has nice hover effect there to indicate that is a link. That said, issue would help us to have the discussion in a better place.

Reply:
I almost implemented the card clickability here to see how you both liked it. It was an easy fix, and I felt that it worked nice and smoothly. But in the end, the selenium-tests prohibited my attempts to make the whole card clickable, they kept breaking. And I thought that I can continue working on those, if we decide to make the card clickable. I agree with Matthew here, that the clickable card with a hover would be nice. 

I'll create an issue for this discussion and/or decision. 

With the changes in this PR, the button has been removed, the image and course name stay clickable. A hover-effect has been added to the course name. Are those good for now?

Reply:
> the button has been removed

I missed that. In that case I think the whole box should be the "button". Thus, I would prefer to keep the butten as is for now in this PR and address the course link side in another issue where we can decide how it should be done. For example, I think we should have similar hover effects as aalto.fi for the activatable box, thus it's not just moving the `a` element.

To conclude, if the button is removed, then the whole box should be the activator, but then ithe behavior must be clear for the user too.

Reply:
I would suggest that this is not the correct call. I highlighted the buttons last week as they have identical link text while pointing to different pages. 

The course name is still formatted as a link, so I don't believe there is a risk of it not being identified as clickable. 

This pull request is a proposal that would fix all currently identified WCAG violations, and your instructions are to re-add one. 

I would strongly advocate either making the whole box clickable _at this point_, or accepting that the button has been removed.

Reply:
This topic was discussed elsewhere, and we decided to go with card sized buttons. Saara will work on it and update the PR.

Reply:
The cards are now clickable apart from the footer as discussed. Hover-effect has been added to text and image to indicate the clickability.